app/career/page.tsx

```diff
@@ -117,6 +117,8 @@ export default function Career() {
                                             href="https://spiral.xyz/#grants"
                                             target="_blank"
                                             rel="noopener noreferrer"
+                                            data-umami-event={`funding-org-clicked`}
```

Comment:
I don't see the event text specific to each link in the Umami dashboard. So I suggest we add the funding source's name to the event itself. The same applies to other funding sources
```suggestion
                                            data-umami-event={`funding-org-spiral-clicked`}
```